ci/jenkins/mellanox/README.md

```diff
@@ -0,0 +1,72 @@
+# Mellanox CI
+
+Mellanox CI is used to test hardware offload in the Antrea project.
+ It deploys Kubernetes with the Antrea image and tests whether the
+ OVS has hardware offload capabilities or not.
+
+## Workflow
+
+The CI tests run on a single node cluster with a ConnectX5 NIC card,
+ the latest stable Kubernetes binaries are built each time the scripts
+ run. In addition to the Kubernetes and the Antrea, the scripts deploy
+ Multus CNI, SRIOV device plugin, and container networking plugins. The
+ versions for the used components are summarized as follow:
+
+* Kubernetes: Latest stable.
+* Antrea: Image built from pull request.
+* Multus: A nightly pulled latest stable Multus image.
+* SRIOV device plugin: A nightly built master image.
```

Comment:
```suggestion
* SRIOV device plugin: A nightly built image from the main branch.
```

Reply:
My bad, forgot about the naming convention. Done